Source/WebCore/css/CSSPseudoSelectors.json

```diff
@@ -24,7 +24,8 @@
         },
         "notes": [
             "Pseudos that start with `-apple-` require a 'settings-flag' field that points to a setting that is disabled to web content by default.",
```

Comment:
I wonder if we actually want to suggest these get never exposed to web content.

Reply:
Adjusted the wording a bit